src/effects/native/flangereffect.cpp

```diff
@@ -175,29 +171,25 @@ void FlangerEffect::processChannel(const ChannelHandle& handle,
     double minManual = kCenterDelayMs - (kMaxLfoWidthMs - width) / 2;
     manual = math_clamp(manual, minManual, maxManual);
 
-    const double width_delta = (width - pState->prev_width) /
-            (numSamples / kChannels);
-    const double width_start = pState->prev_width + width_delta;
+    RampingValue<double> widthRamped(
+            pState->prev_width, width, numSamples / kChannels);
     pState->prev_width = width;
 
-    const double manual_delta = (manual - pState->prev_manual) /
-            (numSamples / kChannels);
-    const double manual_start = pState->prev_manual + manual_delta;
+    RampingValue<double> manualRamped(
+            pState->prev_manual, manual, numSamples / kChannels);
     pState->prev_manual = manual;
 
     CSAMPLE* delayLeft = pState->delayLeft;
     CSAMPLE* delayRight = pState->delayRight;
 
+    QList<int>();
+
     for (unsigned int i = 0; i < numSamples; i += kChannels) {
 
-        CSAMPLE_GAIN mix_ramped = mix_start
-                + mix_delta * i / kChannels;
-        CSAMPLE_GAIN regen_ramped = regen_start
-                + regen_delta * i / kChannels;
-        CSAMPLE_GAIN width_ramped = width_start
-                + width_delta * i / kChannels;
-        CSAMPLE_GAIN manual_ramped = manual_start
-                + manual_delta * i / kChannels;
+        CSAMPLE_GAIN mix_ramped = mixRamped.getNext();
```

Comment:
The variable names are confusing here. There are variables with two different types whose names have the same words but are written slightly differently. Maybe you could rename the "mix" variable above to "mixParameter" and rename "mix_ramped" here to just "mix".